pkgs/by-name/rw/rwpspread/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, libxkbcommon
+, stdenv
+, wayland
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rwpspread";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "0xk1f0";
+    repo = "rwpspread";
+    rev = "v${version}";
+    hash = "sha256-slxsicASZ7JoUnnQf4R3xFB4zgtt4ZOZCU0NcbgBneM=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

```suggestion
    lockFile = "${src}/Cargo.lock";
```
prevents needing to vendor some massive lockfile